docs/file-reference.md

```diff
@@ -161,30 +161,55 @@ name: mariadb
 controller: deployment
 ```
 
+> Example using DeploymentConfig for OpenShift
+
+```yaml
+controller: deploymentconfig
+name: httpd
+replicas: 2
+containers:
+- image: bitnami/nginx
+services:
+- name: httpd
+  type: NodePort
+  ports:
+  - port: 8080
+    targetPort: 80
```

Comment:
one more small change. `bitnami/nginx` is listening in `8080`

Reply:
done.